templates/mammo_form.html

```diff
@@ -33,6 +33,30 @@ <h1>Folder Number {{form.fld_folder_number.data}}</h1>
     });
   }
 
+  function toggleOther(selectorFieldId, otherFieldId) {
+    
+    var selection = $(selectorFieldId).val() == "other";
+    otherField = $(otherFieldId)
+    if(typeof otherField != 'undefined') {
```

Comment:
`if(typeof otherField != 'undefined') { `
changed this to 
`if(typeof otherField != 'other') { `

Reply:
You are right, the problem was that the choice I attached it to did not have an other in it so it wouldn't show. However I have to figure out how to remove any text in the 'other' field on edit to a valid choice...